common/src/main/scala/react/floatingui/Tooltip.scala

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.floatingui
+
+import cats.syntax.all.*
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.feature.ReactFragment
+import japgolly.scalajs.react.vdom.TagOf
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.ui.syntax.all.*
+import lucuma.ui.syntax.all.given
+import org.scalajs.dom
+import org.scalajs.dom.html
+import react.common.ReactFnProps
+import react.common.Style
+import react.floatingui.hooks._
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
+import scala.scalajs.js.|
+
+import js.annotation._
+
+/**
+ * Tooltip base on floating ui see: https://floating-ui.com/docs/react-dom
+ */
+case class Tooltip(trigger: VdomTag, tooltip: VdomNode, placement: Placement = Placement.Top)
+    extends ReactFnProps[Tooltip](Tooltip.component)
+
+object Tooltip {
+  type Props = Tooltip
```

Comment:
Can we make `Props` and `component` private and protected?

Reply:
Are we doing that often?

Reply:
Some of our components have it already, but I think we should.